ADBench/plot_graphs.py

```diff
@@ -29,9 +29,29 @@
 figure_size = (9, 6) if do_plotly else (12, 8)
 fig_dpi = 96
 save_dpi = 144
-colors = ["b", "g", "r", "c", "m", "y"]
-markers = ["*", "+", "s", "^"]
-all_styles = [(c, m) for m in markers for c in colors]
+
+# for tools that have no mapped style
+default_styles = [ (color, "x") for color in "rgbcmyk" ]
+tool_styles = {
+    # C++ tools
+    "Finite": ("b", "*", "C++, Finite"),
+    "FiniteEigen": ("g", "*", "C++, Finite Eigen"),
+    "Manual": ("r", "*", "C++, Manual"),
+    "ManualEigen": ("c", "*", "C++, Manual Eigen"),
+    "ManualEigenVector": ("m", "*", "C++, Manual Eigen Vector"),
+    "Tapenade": ("y", "*", "C, Tapenade"),
+
```

Comment:
Suggest colour = black for all Manual, and then distinguish via marker.  Can you also add linestyle?
colour = Blue for all Finite

Reply:
Do we actually need different linestyles?